Makefile

```diff
@@ -2,9 +2,9 @@ SHORT_NAME := minio
 
 # dockerized development environment variables
 REPO_PATH := github.com/deis/${SHORT_NAME}
-DEV_ENV_IMAGE := quay.io/deis/go-dev:0.20.0
```

Comment:
Why not use quay.io?

Reply:
I have no particular preference for this. quay.io is not very good in our network. I changed it when I was doing E2E test. It has been changed back to quay.io